e2e-tests/test/anvil-apis.test.ts

```diff
@@ -0,0 +1,36 @@
+import { expect } from "chai";
+import { Wallet } from "zksync-web3";
+import { getTestProvider } from "../helpers/utils";
+import { ethers } from "hardhat";
+
+const provider = getTestProvider();
+
+describe("anvil_setBalance", function () {
```

Comment:
What is the reasoning behind creating a new test suite for each test?
Wouldn't it be more logical to create one `describe` block and add an `it` blocks for each endpoint?

Reply:
Well, mostly it was already implemented that way, but it does feel logical to divide the tests by the API - really every API should have multiple `it` blocks (at least 1 success + 1 failure), which can be run together after changing the API...

Reply:
Describe blocks are for the method being tested. This is for readability when there are test failures. Ideally you have multiple "it" blocks within a Describe block to ensure the method functions as expected.

Reply:
that's strange but ok